### PR TITLE
Fix references to moved repos

### PR DESCRIPTION
**Pull request by smaye81**
This fixes the references to repos that were moved to the connectrpc organization (including this one).

### CODE REVIEW

File: packages/connect-query/src/connect-query-key.ts [hunk 1]
```diff
@@ -61,8 +61,5 @@ export const makeConnectQueryKeyGetter =
   (typeName: ServiceType['typeName'], methodInfoName: MethodInfo['name']) =>
   <I extends Message<I>>(
     input?: DisableQuery | PartialMessage<I> | undefined,
-  ): ConnectQueryKey<I> => [
-    typeName,
-    methodInfoName,
-    input === disableQuery || !input ? {} : input,
-  ];
+  ): ConnectQueryKey<I> =>
+    [typeName, methodInfoName, input === disableQuery || !input ? {} : input];
```
Comment (timostamm) [inline packages/connect-query/src/connect-query-key.ts:65]: We should not touch code in this PR, even if it is just formatting. Can you revert?

File: packages/connect-query/src/unary-hooks.test.ts [hunk 2]
```diff
@@ -139,14 +139,11 @@ describe('unaryHooks', () => {
     const input = { sentence: 'ziltoid' } satisfies PartialMessage<SayRequest>;
     const transport = mockEliza();
 
-    const { result } = renderHook(
-      async () => {
-        const { queryFn } = genSay.useQuery(input);
+    const { result } = renderHook(async () => {
```
Comment (timostamm) [inline packages/connect-query/src/unary-hooks.test.ts:142]: We should not touch code in this PR, even if it is just formatting. Can you revert?

File: README.md [hunk 3]
```diff
@@ -491,7 +491,7 @@ Your Protobuf files serve as the primary input to the code generators `protoc-ge
 
 ## What is `Transport`
 
-`Transport` is a regular JavaScript object with two methods, `unary` and `stream`.  See the definition in the Connect-Web codebase [here](https://github.com/bufbuild/connect-web/blob/main/packages/connect-web/src/transport.ts).  `Transport` defines the mechanism by which the browser can call a gRPC-web or Connect backend.  Read more about Transport on the [connect docs](https://connectrpc.com/docs/web/choosing-a-protocol).
+`Transport` is a regular JavaScript object with two methods, `unary` and `stream`.  See the definition in the Connect-Web codebase [here](https://github.com/connectrpc/connect-es/blob/main/packages/connect-web/src/transport.ts).  `Transport` defines the mechanism by which the browser can call a gRPC-web or Connect backend.  Read more about Transport on the [connect docs](https://connectrpc.com/docs/web/choosing-a-protocol).
```
Comment (timostamm) [inline README.md:494]: Can you fix the link to transport.ts?

Comment (smaye81) [reply README.md]: Nice catch. Done.

**Review by timostamm — APPROVED** [review]

**Pull request merged**